module1.js

```diff
@@ -110,6 +126,20 @@ function question7(array1, array2, extraElement) {
 */
 
 function question8(retailPrice, taxAmount) {
+  retailPrice = retailPrice.substring(1);
+  var retailPriceToNumber = parseInt(retailPrice, 10); //coverting retail price to number inorder to multiply.
+  var calculatedTaxOnPrice = taxAmount * retailPriceToNumber;
+  return `£${retailPriceToNumber + calculatedTaxOnPrice}`;
```

Comment:
Great! 
One consideration though - you can also extract the £ symbol from the `retailPrice` variable :)